src/promptflow-evals/CHANGELOG.md

```diff
@@ -5,7 +5,19 @@
 ### Features Added
 - Introduced `JailbreakAdversarialSimulator` for customers who need to do run jailbreak and non jailbreak adversarial simulations at the same time. More info in the README.md in `/promptflow/evals/synthetic/README.md#jailbreak-simulator`
 - Exposed batch evaluation run timeout via `PF_BATCH_TIMEOUT_SEC` environment variable. This variable can be used to set the timeout for the batch evaluation for each evaluator and target separately only, not the entire API call.
+- Support for Tier 1 languages in the simulator.

```

Comment:
From looking online, it doesn't seem like "Tier 1" is a known phrase to describe a specific set of ISO language standards. I think the "1" may refer to the suffix of ISO 639-1, but that list contains 125 languages, not just the ones we have here.

Can we explicitly list the initial covered languages here and link to the ISO documentation: https://www.andiamo.co.uk/resources/iso-language-codes/?